2025-07-09 00:19:17,071 - asyncio - DEBUG - Using proactor: IocpProactor
2025-07-09 00:19:17,071 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-09 00:19:17,074 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-09 00:19:17,082 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-09 00:19:17,082 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-09 00:19:17,092 - __main__ - INFO - Telegram Manager Bot started. Send /menu to interact.
2025-07-09 00:19:17,092 - telegram.ext.ExtBot - DEBUG - Entering: send_message
2025-07-09 00:19:17,100 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-09 00:19:22,105 - httpcore.connection - DEBUG - connect_tcp.failed exception=ConnectTimeout(TimeoutError())
2025-07-09 00:19:22,105 - __main__ - ERROR - Could not send startup message to TELEGRAM_CHAT_ID 7049055170: Timed out
2025-07-09 00:19:22,108 - telegram.ext.ExtBot - DEBUG - Entering: get_me
2025-07-09 00:19:22,108 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-09 00:19:27,107 - httpcore.connection - DEBUG - connect_tcp.failed exception=ConnectTimeout(TimeoutError())
2025-07-09 00:19:27,107 - telegram.ext.Application - DEBUG - This Application is already shut down. Returning.
2025-07-09 00:19:27,110 - __main__ - ERROR - Error in polling: Timed out
